### DISCUSSION TITLE
Please help me with contributions (Song List)

### DISCUSSION THREAD

**Discussion by duonglvtnaist** [NAIST-SE/AttractingContributors#33] [General]
I need help with my anime list.

Please look at this list, it can help you to do it easier ^^!
https://ghibli.fandom.com/wiki/Category:Soundtracks

Come to my fork:
https://github.com/duonglvtnaist/AttractingContributors-Songs

Add to my README file. You should add your README file. 
I will help you to make a contribution. 

I look forward to your contributions.

Thank you!

**Comment by vittunyuta-aom** [comment]
added!!
